### ISSUE TITLE
Automated tests?

### ISSUE THREAD

**Issue by pirogoeth** [dcu/onetouch-ssh#15]
We should attempt to add unit and integration tests for this. It would make pushing new releases easier. But, the only problem I can see is building forked sources since Go likes to get all pull-happy with source trees...

Ideas?

**Comment by dcu** [comment]
yes, tests are very important and it's definitely in my TODO list. I started adding some for the geoip module here: https://github.com/dcu/onetouch-ssh/blob/master/utils/geoip_test.go

I added you as collaborator but let's keep sending PRs :)

**Comment by pirogoeth** [comment]
Awesome! Sounds like a plan :)
